gateway-service-definitions/src/main/resources/services/atlas/2.0.0/rewrite.xml

```diff
@@ -28,48 +28,15 @@
         <rewrite template="{$serviceUrl[ATLAS]}/{**}?{**}"/>
     </rule>
 
-    <rule dir="OUT" name="ATLAS/atlas/outbound/links" pattern="css/{**}">
```

Comment:
How are these resources handled now? does the Atlas UI take care of this? if so wouldn't it be better to create a new version of Knox rules so someone using the un-patched or older Atlas UI has some way to get around this breaking change.

Reply:
we have tested the patch. but make sense to have it as new version 2.1.0. I will update PR in sometime.

Reply:
@moresandeep I have updated PR request.
https://github.com/apache/knox/pull/323/commits/d6cc23776bd3742bdcdc7889e93fbf8d1107aeda

Reply:
With the new changes we need not update this, this is for backwards compatibility which will break with this change.